src/app/blocks/batch-block/batch-block.component.scss

```diff
@@ -0,0 +1,19 @@
+:host(.batch-block-flex) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 10px 20px;
+    overflow: hidden;
```

Comment:
If the `overflow: hidden` property it's just a precaution, for a situation that we've not ran into, isn't it best to not have it there and keep the CSS more minimal? @gsambrotta

Reply:
Yeah, I was also battling with myself to decide whether I should leave it or not. 
Let's remove it. It should not make any difference.